### PR TITLE
[tw] Localize `Containerization` for Traditional Chinese

### PR DESCRIPTION
**Pull request by rosawu159**
### Describe your changes

Origin content from https://glossary.cncf.io/containerization/

### Related issue number or link (ex: `resolves #issue-number`)

resolves https://github.com/cncf/glossary/issues/2478

### Checklist before opening this PR (put `x` in the checkboxes)
- [V] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [V] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/containerization.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+容器化是將一個應用程式及其相關依賴項打包成一個容器映像的過程。容器建構過程需要遵循[開放容器倡議](https://opencontainers.org/)（OCI）標準。只要輸出是符合此標準的容器映像，使用哪個容器化工具都不重要。
+
+## 解決的問題 {#problem-it-addresses}
+
+在容器變得普遍之前，組織依賴虛擬機器來在單一[裸機]((/zh-tw/bare-metal-machine/))上協調多個應用程式。虛擬機器比容器大得多，並且需要虛擬機器管理程式才能運作。由於這些較大的虛擬機器模板的儲存、備份和傳輸，建立虛擬機器模板的速度也很慢。此外，虛擬機器可能會受到配置漂移的影響，這違反了[不變性原則](/zh-tw/immutable-infrastructure/)。
+
+## 如何幫助我們 {#how-it-helps}
+
+容器映像檔輕巧（與傳統虛擬機器不同），容器化過程需要一個列出依賴項目的檔案。這個檔案可以進行版本控制，並且可以自動化建置過程，使組織可以專注於其他重要事務，而自動化流程則負責建置。容器映像檔以唯一識別碼存儲，與其確切的內容和配置相關聯。由於容器被安排和重新安排，它們始終被重置為初始狀態，這消除了配置漂移的問題。
```
Comment (HungWeiChiu) [inline content/zh-tw/containerization.md:18]: 建議按照原文分段落，不需要把所有內容變成一行，這樣 review 過程會比較難針對單一句子留言

Comment (HungWeiChiu) [inline content/zh-tw/containerization.md:18]: 配置 -> 似乎是簡體中文用法？

Comment (HungWeiChiu) [inline content/zh-tw/containerization.md:18]: 不確定我們其他翻譯是不是都用 安排 來翻譯 "schedule"

File: content/zh-tw/containerization.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title:  容器化
+status: Completed
+category: 技術
+tags: ["基本原理", "基礎設施", "方法論"]
+---
+
+## 是什麼 {#what-it-is}
+
```
Comment (hwchiu) [inline content/zh-tw/containerization.md:9]: 內容參照原文分多行，全部擠成一行對於 review 會有困難

File: content/zh-tw/containerization.md (hunk 1 above)
Comment (hwchiu) [inline content/zh-tw/containerization.md:18]: 不確定我們其他翻譯是不是都用 安排 來翻譯 "schedule"

File: content/zh-tw/containerization.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+容器化是將一個應用程式及其相關依賴項打包成一個容器映像的過程。容器建構過程需要遵循[開放容器倡議](https://opencontainers.org/)（OCI）標準。只要輸出是符合此標準的容器映像，使用哪個容器化工具都不重要。
+
+## 解決的問題 {#problem-it-addresses}
+
+在容器變得普遍之前，組織依賴虛擬機器來在單一[裸機]((/zh-tw/bare-metal-machine/))上協調多個應用程式。虛擬機器比容器大得多，並且需要虛擬機器管理程式才能運作。由於這些較大的虛擬機器模板的儲存、備份和傳輸，建立虛擬機器模板的速度也很慢。此外，虛擬機器可能會受到配置漂移的影響，這違反了[不變性原則](/zh-tw/immutable-infrastructure/)。
```
Comment (hwchiu) [inline content/zh-tw/containerization.md:14]: 配置似乎是簡體中文用語？ 要確認一下

File: content/zh-tw/containerization.md (hunk 1 above)
Comment (rosawu159) [reply content/zh-tw/containerization.md:18]: /zh-tw/kubernetes/ 對schedule的翻譯是『排程』，會修正為此翻譯

File: content/zh-tw/containerization.md (hunk 3 above)
Comment (rosawu159) [reply content/zh-tw/containerization.md:14]: 參考了[Oracle](https://www.oracle.com/tw/security/cloud-security/what-is-cloud-security/)和[Azure](https://learn.microsoft.com/zh-tw/azure/well-architected/devops/principles)的說法，會修正為 組態漂移

**Review by hwchiu — APPROVED** [review]
LGTM

File: content/zh-tw/containerization.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## 是什麼 {#what-it-is}
+
+容器化是將一個應用程式及其相關依賴項打包成一個容器映像的過程。
+容器建構過程需要遵循[開放容器倡議](https://opencontainers.org/)（OCI）標準。
+只要輸出是符合此標準的容器映像，使用哪個容器化工具都不重要。
+
+## 解決的問題 {#problem-it-addresses}
+
+在容器變得普遍之前，組織依賴虛擬機器在單一[裸機]((/zh-tw/bare-metal-machine/))上協調多個應用程式。
```
Comment (pichuang) [inline content/zh-tw/containerization.md:16]:
```suggestion
在容器變得普遍之前，組織依賴虛擬機器在單一[裸機](/zh-tw/bare-metal-machine/)上協調多個應用程式。
```

File: content/zh-tw/containerization.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+title:  容器化
+status: Completed
+category: 技術
+tags: ["基本原理", "基礎設施", "方法論"]
+---
+
+## 是什麼 {#what-it-is}
+
+容器化是將一個應用程式及其相關依賴項打包成一個容器映像的過程。
+容器建構過程需要遵循[開放容器倡議](https://opencontainers.org/)（OCI）標準。
```
Comment (pichuang) [inline content/zh-tw/containerization.md:11]:
```suggestion
容器建構過程需要遵循[開放容器倡議組織](https://opencontainers.org/)（OCI）標準。
```

File: content/zh-tw/containerization.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+只要輸出是符合此標準的容器映像，使用哪個容器化工具都不重要。
+
+## 解決的問題 {#problem-it-addresses}
+
+在容器變得普遍之前，組織依賴虛擬機器在單一[裸機]((/zh-tw/bare-metal-machine/))上協調多個應用程式。
+虛擬機器比容器大得多，並且需要虛擬機器管理程式才能運作。
+由於這些較大的虛擬機器模板的儲存、備份和傳輸，建立虛擬機器模板的速度也很慢。
```
Comment (pichuang) [inline content/zh-tw/containerization.md:18]:
```suggestion
由於這些較大的虛擬機器樣板的儲存、備份和傳輸，建立虛擬機器模板的速度也很慢。
```

File: content/zh-tw/containerization.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+---
+title:  容器化
```
Comment (pichuang) [inline content/zh-tw/containerization.md:2]:
```suggestion
title: 容器化
```

File: content/zh-tw/containerization.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+容器化是將一個應用程式及其相關依賴項打包成一個容器映像的過程。
+容器建構過程需要遵循[開放容器倡議](https://opencontainers.org/)（OCI）標準。
+只要輸出是符合此標準的容器映像，使用哪個容器化工具都不重要。
+
+## 解決的問題 {#problem-it-addresses}
+
+在容器變得普遍之前，組織依賴虛擬機器在單一[裸機]((/zh-tw/bare-metal-machine/))上協調多個應用程式。
+虛擬機器比容器大得多，並且需要虛擬機器管理程式才能運作。
+由於這些較大的虛擬機器模板的儲存、備份和傳輸，建立虛擬機器模板的速度也很慢。
+此外，虛擬機器可能會受到組態漂移的影響，這違反了[不變性原則](/zh-tw/immutable-infrastructure/)。
```
Comment (pichuang) [inline content/zh-tw/containerization.md:19]:
```suggestion
此外，虛擬機器可能會受到組態漂移的影響，這違反了[不可變原則](/zh-tw/immutable-infrastructure/)。
```

**Comment by pichuang** [comment]
@rosawu159 請將 9 個 commits 使用 git squash 收成 1 個，即可合併

File: content/zh-tw/containerization.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+## 是什麼 {#what-it-is}
+
+容器化是將一個應用程式及其相關依賴項打包成一個容器映像的過程。
+容器建構過程需要遵循[開放容器倡議組織](https://opencontainers.org/)（OCI）標準。
+只要輸出是符合此標準的容器映像，使用哪個容器化工具都不重要。
+
+## 解決的問題 {#problem-it-addresses}
+
+在容器變得普遍之前，組織依賴虛擬機器在單一[裸機](/zh-tw/bare-metal-machine/)上協調多個應用程式。
```
Comment (johnlinp) [inline content/zh-tw/containerization.md:16]: 我覺得這句如果改成這樣會比較通順，不知道你覺得如何呢？

```suggestion
在容器變得普遍之前，組織需要使用在單一[裸機](/zh-tw/bare-metal-machine/)上的多個虛擬機器來協調多個應用程式。
```

原文:

> Before containers became prevalent, organizations relied on virtual machines (VMs) to orchestrate multiple applications on a single [bare-metal machine](https://glossary.cncf.io/bare-metal-machine/).

File: content/zh-tw/containerization.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+虛擬機器比容器大得多，並且需要虛擬機器管理程式才能運作。
+由於這些較大的虛擬機器樣板的儲存、備份和傳輸，建立虛擬機器樣板的速度也很慢。
```
Comment (johnlinp) [inline content/zh-tw/containerization.md:18]: 這一句我也建議這樣改會比較通順：

```suggestion
由於需要儲存、備份和傳輸這些較大的虛擬機器樣板，建立虛擬機器樣板的速度也很慢。
```

原文:

> Due to the storage, backup, and transfer of these larger VM templates, creating the VM templates is also slow.

File: content/zh-tw/containerization.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
+---
+title: 容器化
+status: Completed
+category: 技術
+tags: ["基本原理", "基礎設施", "方法論"]
+---
+
+## 是什麼 {#what-it-is}
+
+容器化是將一個應用程式及其相關依賴項打包成一個容器映像的過程。
```
Comment (johnlinp) [inline content/zh-tw/containerization.md:10]: 我建議把 "container image" 一律翻譯成 "容器映像檔" ，就像你下面翻的一樣。

File: content/zh-tw/containerization.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+容器建構過程需要遵循[開放容器倡議組織](https://opencontainers.org/)（OCI）標準。
+只要輸出是符合此標準的容器映像，使用哪個容器化工具都不重要。
```
Comment (johnlinp) [inline content/zh-tw/containerization.md:12]: 這邊也是建議把 "容器映像" 改為 "容器映像檔" ，謝謝。

**Review by ydFu — CHANGES_REQUESTED** [review]
Please compress redundant commits into one

**Review by ydFu — APPROVED** [review]
Thanks for compressing git commit

File: content/zh-tw/containerization.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
+<<<<<<< HEAD
+=======
```
Comment (ydFu) [inline content/zh-tw/containerization.md:2]:
```suggestion
<<<<<<< HEAD
=======
```
delete this line

**Review by ydFu — CHANGES_REQUESTED** [review]

File: content/zh-tw/containerization.md [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+
+## 是什麼 {#what-it-is}
+
+容器化是將一個應用程式及其相關依賴項打包成一個容器映像檔的過程。
+容器建構過程需要遵循[開放容器倡議組織](https://opencontainers.org/)（OCI）標準。
+只要輸出是符合此標準的容器映像檔，使用哪個容器化工具都不重要。
+
+## 解決的問題 {#problem-it-addresses}
+
+在容器變得普遍之前，組織需要使用在單一[裸機](/zh-tw/bare-metal-machine/)上的多個虛擬機器來協調多個應用程式。
```
Comment (ydFu) [inline content/zh-tw/containerization.md:16]:
```suggestion
在容器變得普遍之前，組織依賴虛擬機器(VM)在單一[裸機](/zh-tw/bare-metal-machine/)上協調多個應用程式。
```

en
```
Before containers became prevalent, organizations relied on virtual machines (VMs) to 
orchestrate multiple applications on a single [bare-metal machine](/bare-metal-machine/).
```

File: content/zh-tw/containerization.md [hunk 15]
```diff
@@ -0,0 +1,26 @@
...
+虛擬機器比容器大得多，並且需要虛擬機器管理程式才能運作。
+由於需要儲存、備份和傳輸這些較大的虛擬機器樣板，建立虛擬機器樣板的速度也很慢。
```
Comment (ydFu) [inline content/zh-tw/containerization.md:18]:
```suggestion
由於需要較大儲存、備份和傳輸的虛擬機器樣板，建立虛擬機器樣板的速度也很慢。
```

**Review by ydFu — CHANGES_REQUESTED** [review]
nit

**Review by johnlinp — APPROVED** [review]
感謝!

**Comment by ydFu** [comment]
@rosawu159 
May I inquire if the current pull request is still active, and if there are any technical challenges that need to be addressed?

**Comment by rosawu159** [comment]
@ydFu Everything is fine. All technical challenges have been resolved in the ongoing conversation, please check.

**Pull request merged**